### DISCUSSION TITLE
Question: Does XR also work with Laravel?

### DISCUSSION THREAD

**Discussion by slakbal** [chevere/xr#73] [Ideas]
Hi There, 

This looks really awesome, quick question will it also work with Laravel?

Thanks!

**Comment by rodber** [comment]
Hi @slakbal

XR Debug works in any PHP 8 codebase.

At this time it doesn't provide _extra_ support for Laravel or any other framework/cms (as tailored methods for these) but there's a layer for adding custom inspectors: https://xr-docs.chevere.org/developer/custom-inspectors.html

**Comment by davisngl** [comment]
I have nothing to add this and you can either delete or mark my comment as a spam,

but I really appreciate the work you have put in this. Spatie is lowering their standards with their package maintenance + overpricing glorified `dd` tool that's not even properly cross-compatible without bloat.

Give them run for their money!

**Reply by rodber** [reply]
> I have nothing to add this and you can either delete or mark my comment as a spam,
> 
> but I really appreciate the work you have put in this. Spatie is lowering their standards with their package maintenance + overpricing glorified `dd` tool that's not even properly cross-compatible without bloat.
> 
> Give them run for their money!

@davisngl 

My pleasure. By all means I encourage you to elaborate a list with all the Laravel dev related tools that are being gate-keep/overpriced by other developers/companies. I have no remorse in doing it better and free.

**Comment by rodber** [comment]
> Hi There,
> 
> This looks really awesome, quick question will it also work with Laravel?
> 
> Thanks!

Hello @slakbal

Just released v1.0.0 for xrDebug Laravel integration: https://github.com/xrdebug/laravel hope you can give it a try. Sorry if it took that long.

**Reply by slakbal** [reply]
Thanks will take a look!!!
